validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -4,6 +4,86 @@
 import StdlibUnittest
 import StdlibCollectionUnittest
 
+// Tests
+
+struct SubscriptGetTest {
+  // SubscriptGetTest operates on a `(end - start)` sized buffer containing
+  // monotonically increasing integers from `start` to `end - 1`.
+  static let start = 20
+  static let end = 100
+  let range: Range<Int>
+  /// The values that should be expected by slicing the UBP, or `nil` if the
+  /// test is expected to crash.
+  let expectedValues: [Int]?
+  let loc: SourceLoc
+
+  static var elementCount = (end - start)
+
+  /// Create and populate an `UnsafeBufferPointer` for use with unit tests.
+  /// PRECONDITION: `memory` must be allocated with space for
+  /// `SubscriptGetTest.elementCount` elements. 
+  func createUnsafeBufferPointer(from memory: UnsafeMutablePointer<OpaqueValue<Int>>) 
+    -> UnsafeBufferPointer<OpaqueValue<Int>> 
+  {
+    for i in SubscriptGetTest.start..<SubscriptGetTest.end {
+      memory[i] = OpaqueValue(i)  
+    }
+    return UnsafeBufferPointer(start: memory, 
+                               count: SubscriptGetTest.elementCount)
+  }
+
+  /// Create an populate an `UnsafeMutableBufferPointer` for use with unit
+  /// tests.
+  /// PRECONDITION: `memory` must be allocated with space for
+  /// `SubscriptGetTest.elementCount` elements.
+  func createUnsafeMutableBufferPointer(from memory: UnsafeMutablePointer<OpaqueValue<Int>>) 
```

Comment:
These 2 functions: `createUnsafeBufferPointer` and `createUnsafeMutableBufferPointer` are GYBbable.